src/styles/utils/normalize.scss

```diff
@@ -0,0 +1,11 @@
+* {
+  box-sizing: border-box;
+}
+
+html {
+  font-family: "Roboto", sans-serif;
+}
+
```

Comment:
I would rather set font-family for page block, not in normalize file.

Reply:
thanks, it does seem like a better practice